frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -27,7 +30,12 @@ import { useContext, useEffect, useState } from "react";
 import { SearchContext } from "../../../shared/contexts";
 import { baseBackendUrl } from "../../../shared/urls";
 
-// schema: https://github.com/jquense/yup?tab=readme-ov-file#stringurlmessage-string--function-schema
+const Root = styled('div')(({ theme }) => ({
+  width: '100%',
+  "& > * + *": {
```

Comment:
curious about what this does and why we can't just do something like `marginTop: '3rem'`

Reply:
Also, I don't even think we need the Root styling here to be honest. I noticed that with it the form is a bit off-center.

Reply:
I'm with you, removed the root styling since it's not necessary.